kolibri/plugins/epub_viewer/assets/src/views/SettingsSideBar.vue

```diff
@@ -177,6 +323,21 @@
         context:
           "The EPUB reader allows learners to set the background of the reader to different shades of colors using the 'Theme' option. In this case it can be set to black.",
       },
+      setCustomTheme: {
+        message: 'Set custom theme {themeName}',
```

Comment:
For user generated text, we usually wrap the text in quotes. So this (and the others for the other CRUD actions) would be 

"Set custom theme '{themeName}'" (which I think will manage the quotes without needing to escape them)

Reply:
done. applied for set, delete, and edit theme strings.